packages/ui/src/tailwind.cjs

```diff
@@ -0,0 +1,97 @@
+const colors = {
```

Comment:
Can't this be a .mjs file? Just by preference

Reply:
This file is imported by the `tailwind.config.js` of the app, as commonjs which is what Tailwind allows. I could add another entrypoint with different config to SWC to turn this file from `.mjs` into `.cjs` but that sounds like an unnecessary step.